api/v1/lib/httpcli/httpsched/httpsched.go

```diff
@@ -96,6 +97,14 @@ func MaxRedirects(mr int) Option {
 	}
 }
 
+func AllowReconnection(v bool) Option {
```

Comment:
mind writing a minimal comment for this func that explains how it effectively alters the behavior of the client?

Reply:
i'm going to add comments in a subsequent PR